all.jsonnet

```
@@ -62,6 +62,16 @@ local s = t.store + t.store.withVolumeClaimTemplate + t.store.withServiceMonitor
   },
 };
 
+local swm = t.store + t.store.withVolumeClaimTemplate + t.store.withServiceMonitor + t.store.withMemcachedIndexCache + commonConfig + {
```

Let's add a few line breaks for readability.
```suggestion
local swm = 
  t.store + 
  t.store.withVolumeClaimTemplate + 
  t.store.withServiceMonitor + 
  t.store.withMemcachedIndexCache + 
  commonConfig + 
  {
```